Automattic/wp-calypso
# Launchpad: Add navigator data store

Related to https://github.com/Automattic/jetpack/pull/32963

## Proposed Changes

* Adds the data stores for the new Launchpad Navigator component. This will be responsible for fetching the current Launchpad navigator status and updating the current checklist.

## Testing Instructions



* For now, we only have the use case for setting the active task list
* To test it, apply this branch or use the calypso live link
* Make sure you are sandboxed and have the https://github.com/Automattic/jetpack/pull/32964 changes applied
* Create a new site or use an existing one that has a post-launch Launchpad
* Click on a task. It should make a `PUT` request to `/launchpad/navigator`

PS: We are not yet hydrating the initial value. We'll do it after the https://github.com/Automattic/wp-calypso/pull/81575, which will prevent the `/launchpad/navigator` from being called more than once.

## Pre-merge Checklist
